invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/DisplayVerifiedCommunity.js

```diff
@@ -0,0 +1,53 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2024 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Label, Image } from "semantic-ui-react";
+
+export const DisplayVerifiedCommunity = ({ communities }) => {
+  const displayCommunity = (parentCommunities) => {
+    const brandedCommunityId = parentCommunities?.default;
+    const communities = parentCommunities?.entries;
+    let displayCommunity = null;
+
+    if (communities) {
+      communities.map((community) => {
+        if (community.id === brandedCommunityId && community?.theme) {
+          displayCommunity = community;
+        }
+        return displayCommunity;
+      });
+    }
+
+    return displayCommunity;
+  };
+
+  const verifiedCommunity = displayCommunity(communities);
```

Comment:
```suggestion
  const verifiedCommunity = filterBrandedCommunity(communities);
```

assuming from the code branded == verified at the moment.